apps/router-demo/router-host-2000/src/App.tsx

```diff
@@ -1,6 +1,6 @@
 import { useRef, useEffect, ForwardRefExoticComponent } from 'react';
 import { Route, Routes, useLocation } from 'react-router-dom';
-import { init, loadRemote } from '@module-federation/enhanced/runtime';
+import { init, loadRemote } from '@module-federation/rsbuild-plugin/runtime';
```

Comment:
Can we import from `@module-federation/enhanced/runtime`? This is consistent with Rspack / webpack projects and avoids differences in runtime code